src/qt/rpcconsole.cpp

```diff
@@ -840,38 +841,41 @@ void RPCConsole::clear(bool keep_prompt)
 
     // Set default style sheet
     QFontInfo fixedFontInfo(GUIUtil::fixedPitchFont());
+    std::string _style(" \
+                table { } \
+                td.time { color: #808080; font-size: %2; padding-top: 3px; } \
+                td.message { font-family: %1; font-size: %2; white-space:pre-wrap; } \
+                td.cmd-request { color: #006060; } \
+                td.cmd-error { color: red; } ");
+    _style.append(
+                networkStyle->getSecWarningStyle().toStdString());
+    _style.append(
+                ("b { color: #006060; } "));
     ui->messagesWidget->document()->setDefaultStyleSheet(
-        QString(
-                "table { }"
-                "td.time { color: #808080; font-size: %2; padding-top: 3px; } "
-                "td.message { font-family: %1; font-size: %2; white-space:pre-wrap; } "
-                "td.cmd-request { color: #006060; } "
-                "td.cmd-error { color: red; } "
-                ".secwarning { color: red; }"
-                "b { color: #006060; } "
+        QString(_style.c_str()
             ).arg(fixedFontInfo.family(), QString("%1pt").arg(consoleFontSize))
         );
 
     static const QString welcome_message =
-        /*: RPC console welcome message.
-            Placeholders %7 and %8 are style tags for the warning content, and
-            they are not space separated from the rest of the text intentionally. */
```

Comment:
Why is this removed?

Reply:
Hi. The `RPC console welcome message` line is removed because this line:
https://github.com/bitcoin-core/gui/blob/d67330d11245b11fbdd5e2dd5343ee451186931e/src/qt/rpcconsole.cpp#L856
is useless comment, as 'welcome message' is already the name of the string, and 'RPC console' is already the name of the class.

The other two lines:
https://github.com/bitcoin-core/gui/blob/d67330d11245b11fbdd5e2dd5343ee451186931e/src/qt/rpcconsole.cpp#L857-L858
are useful comment, and it's moved to somewhere else by my rev https://github.com/bitcoin-core/gui/commit/0c85f16b8b0bdf7387b1fd2792448863efdaf97f.
**I'm sorry it's my fault that is confusing**, because is a change that should had happened before, in my rev https://github.com/bitcoin-core/gui/pull/387/commits/0885b5068edd1753876d66fba272527add22bc3c,
not where it is now.
If this were ever to be merged, I would move that change, that comment movement, to the rev were it belongs.

I hope I have explained myself clearly enough. ^^

Cheers,

Reply:
> is useless comment, as 'welcome message' is already the name of the string, and 'RPC console' is already the name of the class.

This is a [translator comment](https://doc.qt.io/qt-5/i18n-source-translation.html#translator-comments) dedicated to provide a context to translators on Transifex.com.

Reply:
> > is useless comment, as 'welcome message' is already the name of the string, and 'RPC console' is already the name of the class.
> 
> This is a [translator comment](https://doc.qt.io/qt-5/i18n-source-translation.html#translator-comments) dedicated to provide a context to translators on Transifex.com.

Aha, thanks for showing me. I had noticed the colon when coding,
it felt quite weird and I thought it should be there for some reason,
just I didn't know it would fire back at me this soon.

Cheers,

> 
>     the mission becomes to make the tools hapy
> 
>                         laanwj, 2021-08-02T11:16:25, #bitcoin-core-dev